modern/src/map/core/MapView.js

```diff
@@ -78,6 +78,11 @@ const MapView = ({ children }) => {
   const [activeMapStyles] = usePersistedState('activeMapStyles', ['locationIqStreets', 'osm', 'carto']);
   const [defaultMapStyle] = usePersistedState('selectedMapStyle', 'locationIqStreets');
   const mapboxAccessToken = useAttributePreference('mapboxAccessToken');
+  const maxZoom = useAttributePreference('web.maxZoom') || 16;
```

Comment:
The only part that I don't like is the hardcoded default value.

Reply:
I don't think there should be any. Or there might be some depending on the map layer.